src/views/PeopleSearch/index.js

```diff
@@ -90,55 +90,51 @@ const noResultsCard = (
   </Grid>
 );
 
-const peopleSearchHeader = (
-  <Media query="(min-width: 960px)">
-    {(matches) =>
-      matches ? (
-        <div style={styles.headerStyle}>
-          <Grid container direction="row" alignItems="center">
-            <Grid item xs={1} />
-            <Grid item xs={2}>
-              <Typography variant="body2" style={styles.headerStyle}>
-                FIRST NAME
-              </Typography>
-            </Grid>
-            <Grid item xs={2}>
-              <Typography variant="body2" style={styles.headerStyle}>
-                LAST NAME
-              </Typography>
-            </Grid>
-            <Grid item xs={2}>
-              <Typography variant="body2" style={styles.headerStyle} noWrap>
-                DESCRIPTION
-              </Typography>
-            </Grid>
-            <Grid item xs={2}>
-              <Typography variant="body2" style={styles.headerStyle}>
-                CLASS/JOB TITLE
-              </Typography>
-            </Grid>
-            <Grid item xs={2}>
-              <Typography variant="body2" style={styles.headerStyle}>
-                @GORDON.EDU
-                <br />
-                MAIL LOCATION
-              </Typography>
-            </Grid>
-          </Grid>
-        </div>
-      ) : (
-        <div style={styles.headerStyle}>
-          <Grid container direction="row" justify="center">
-            <Grid item>
-              <Typography variant="body2" style={styles.headerStyle}>
-                RESULTS
-              </Typography>
-            </Grid>
-          </Grid>
-        </div>
-      )
-    }
-  </Media>
+const peopleSearchHeaderDesktop = (
+  <div style={styles.headerStyle}>
+    <Grid container direction="row" alignItems="center">
+      <Grid item xs={1} />
+      <Grid item xs={2}>
+        <Typography variant="body2" style={styles.headerStyle}>
+          FIRST NAME
+        </Typography>
+      </Grid>
+      <Grid item xs={2}>
+        <Typography variant="body2" style={styles.headerStyle}>
+          LAST NAME
+        </Typography>
+      </Grid>
+      <Grid item xs={2}>
+        <Typography variant="body2" style={styles.headerStyle} noWrap>
+          DESCRIPTION
+        </Typography>
+      </Grid>
+      <Grid item xs={2}>
+        <Typography variant="body2" style={styles.headerStyle}>
+          CLASS/JOB TITLE
+        </Typography>
+      </Grid>
+      <Grid item xs={2}>
+        <Typography variant="body2" style={styles.headerStyle}>
+          @GORDON.EDU
+          <br />
+          MAIL LOCATION
+        </Typography>
+      </Grid>
+    </Grid>
+  </div>
+);
+
+const peopleSearchHeaderMobile = (
+  <div style={styles.headerStyle}>
+    <Grid container direction="row" justify="center">
+      <Grid item>
+        <Typography variant="body2" style={styles.headerStyle}>
+          RESULTS
+        </Typography>
+      </Grid>
+    </Grid>
+  </div>
 );
 
 class PeopleSearch extends Component {
```

Comment:
I think we should refactor this to a functional component while we're touching these files.

Reply:
This is a smart line of thinking, however, even Cam was hesitant to begin that task, at least now. For the future I think this is a good goal, but these files are quite complicated when it comes to the react tools they use so converting it to a functional component would be a longer process than we have time for currently.